src/redux/initialState.ts

```diff
@@ -72,6 +72,7 @@ const initialAppState: AppState = {
     searchHistory: electronStore.get('appConfig.recentSearch') || [],
     currentMatch: null,
   },
+  supportedExtensions: ['.txt', '.md', '.html', '.json'],
```

Comment:
these are the basic extensions I can see we might want to edit, please advice any other we can start with?